elixir/advanced-guides/sqlite3.html.md

```diff
@@ -0,0 +1,252 @@
+---
+title: SQLite3
+layout: framework_docs
+objective: This guide shows you how to use SQLite3 as your database
+status: beta
+---
+
+While Elixir applications on [Fly.io](https://fly.io) normally run on Postgres databases, you can
+choose to run them on [SQLite3](https://www.sqlite.org/index.html). This guide will assume you have an already setup 
+and configured Phoenix Application using [ecto_sqlite3](https://github.com/elixir-sqlite/ecto_sqlite3) running locally.
+
+To make this work, you will need to place your databases on persistent [Volumes](https://www.sqlite.org/index.html)
+as your deployment image will get overwritten the next time you deploy.
+
+Volumes are limited to one host, this currently means that fly.io hosted Elixir applications that use
+SQLite3 for their database can't be deployed to multiple regions.
+
+But if you are okay using beta software, [LiteFS](/docs/litefs) could work for multi-region sync, check it out! But this guide is going to assume you have one node and one volume.
+
+This guide is heavily influenced by the [guide from](https://gist.github.com/mcrumm/98059439c673be7e0484589162a54a01) Phoenix Core Team Member, Michael Crumm.
+
+Following are the steps required to make this work:
+
+## Create volume
+
+```cmd
+fly volumes create name
+```
+
+Replace `name` with your desired volume name.  Only alphanumeric characters an
+underscores are allowed in names.
+
+Optionally, you may specify the size of the volume, in gigabytes, by adding a `--size int` argument.
+The default volume size is 3 gigabytes.
+
+## Mount and Prep for Deployment
+
+Add the following to your `fly.toml`, once again replacing the name with what you selected, this
+time in two places:
+
+```
+[mounts]
+  source="name"
+  destination="/mnt/name"
+```
+
+Next move remove the release step from the deploy section of your `fly.toml` and add a DATABASE_PATH variable
+
+```diff
+[env]
++ DATABASE_PATH = "/mnt/name/name.db"
+
+-[deploy]
+-  release_command = "/app/bin/migrate"
+```
+
+This step is required because a volume may not be ready once your application release runs, so to fix this we need to run Migrations on 
+application start, this can be accomplished by adding the following line to your `lib/name/application.ex:
+
+```diff
+  @impl true
+  def start(_type, _args) do
++ . Name.Release.migrate()
+    children = [
+```
+
+## Deploy
+
+These changes can be deployed using `fly deploy`.
```

Comment:
I think this is the only command you ask people to run where it doesn't get a code block. I'd consider using one for consistency.